### PR TITLE
docs: update harvesting doc

### PR DESCRIPTION
**Pull request by szymonlopaciuk**
Signed-off-by: Szymon Łopaciuk <szymon@lopaciuk.eu>

## Checklist:

- [ ] I have all the information that I need (if not, move to `RFC` and look for it).
- [ ] I linked the related issue(s) in the corresponding commit logs.
- [ ] I wrote [good commit log messages](https://github.com/torvalds/subsurface-for-dirk/blob/5f15ad5a86ada3c5e574041a5f9d85235322dabb/README#L92-L119).
- [ ] My code follows the code style of this project.
- [ ] I've added any new docs if API/utils methods were added.
- [ ] I have updated the existing documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/harvesting.rst [hunk 1]
```diff
@@ -34,6 +34,11 @@ This document specifies how to harvest records into your system.
 
 If you are going to run harvesting workflows which needs prediction models such as the CORE guessing, keyword extraction, and plot extraction you may need to install some extra packages.
 
+.. warning::
+
+    These services are not Dockerized, so you will have to do that yourself if the need arises.
+    Below instructions are only applicable if you're running inspire locally without Docker.
+
```
Comment (david-caro) [inline docs/harvesting.rst]: What services? Beard and Magpie? It's not clear to me from this phrase :/, can you clarify?

(everything else is, scrapyd/web/worker...)

File: docs/harvesting.rst [hunk 2]
```diff
@@ -130,3 +152,35 @@ Via inspirehep cli:
 .. code-block:: bash
```
[the remark this answers is not in the document]
Comment (szymonlopaciuk) [reply docs/harvesting.rst]: Done :+1:

**Comment by szymonlopaciuk** [comment]
Depends on inspirehep/hepcrawl#249

**Comment by szymonlopaciuk** [comment]
ci merge please

**Pull request merged**